#scan_topic: /base_laser_rear/scan
scan_topic: /scan_unified
transform_link: base_link
local_map_topic: /move_base/global_costmap/costmap

#local_map_topic: /map

leg_radius: 0.1
min_observations: 3
min_predictions: 3
max_dist_btw_legs: 0.8
z_coordinate: 0.0
vel_stance_threshold: 0.47
vel_swing_threshold: 0.93
state_dimensions: 6
clusterTolerance: 0.07
minClusterSize: 3
maxClusterSize: 100
max_nn_gating_distance: 1.0
occluded_dead_age: 10
variance_observation: 0.25
min_dist_travelled: 0.25
max_cov: 0.81
#in_free_space_threshold: 0.06
in_free_space_threshold: 0.3





#history_size: 5